# define deep sensor MS5837_30B message format
# Header
Header header
# Pressure
float32 psr_mbar
float32 psr_atm
float32 psr_Pa
# temperature
float32 temp_C
float32 temp_F
float32 temp_K
# depth
float32 depth_fresh
float32 depth_salt
# altitude
float32 altitude